src/lib/components/lines-and-dots/event-sort-filter.svelte

```diff
@@ -57,7 +57,7 @@
   <MenuButton
     active={!!value}
     hasIndicator
-    variant="primary"
+    variant="table-header"
```

Comment:
We don't need to update this component since it isn't currently being used anywhere. It also looks like it's part of the lines and dots and so it likely wouldn't be in a table anyways.

```suggestion
```